spec/components/grid-item-spec.js

```diff
@@ -0,0 +1,81 @@
+import createComponent from '../helpers/createComponent.js';
+import GridItem, { getFormattedName } from '../../src/renderer/components/grid-item.jsx';
+import generatorsList from '../fixtures/generators-list.js';
+
+
+function mockContent() {
+  const contentDiv = window.document.createElement('div');
+  contentDiv.id = 'content';
+  contentDiv.style.height = '800px';
+  window.document.body.appendChild(contentDiv);
+}
```

Comment:
agreed but I really want to keep the scope of changes to only **introducing tests** as this is priority number 1 in my opinion :blush: 

we should revisit and fix it properly in the future :+1: